### DISCUSSION TITLE
"sqlmodel" is not defined (in alembic version files)

### DISCUSSION THREAD

**Discussion by daudln** [fastapi/sqlmodel#901] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
"""empty message

Revision ID: bd8e5185e36b
Revises: 
Create Date: 2024-04-23 08:50:34.102629

"""
from typing import Sequence, Union

from alembic import op
import sqlalchemy as sa
# missing import sqlmodel.sql.sqltypes

# revision identifiers, used by Alembic.
revision: str = 'bd8e5185e36b'
down_revision: Union[str, None] = None
branch_labels: Union[str, Sequence[str], None] = None
depends_on: Union[str, Sequence[str], None] = None


def upgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.create_table('permission',
    sa.Column('id', sa.Integer(), nullable=False),
    sa.Column('unique_id', sqlmodel.sql.sqltypes.GUID(), nullable=True),
    sa.Column('name', sqlmodel.sql.sqltypes.AutoString(), nullable=False),
    sa.Column('code', sqlmodel.sql.sqltypes.AutoString(), nullable=False),
    sa.PrimaryKeyConstraint('id'),
    sa.UniqueConstraint('unique_id')
    )
    # ...
```

### Description

Every time I run 
```sh
alembic revision --autogenerate
```
In versions files, `sqlmodel` is not automatically imported. Which make me navigate to version file and add this line of code in input

```py
import sqlmodel.sql.sqltypes
```

### Operating System

Linux

### Operating System Details

Ubuntu 22.04

### SQLModel Version

0.0.16

### Python Version

3.10.12

**Comment by lanbaoshen** [comment, marked as answer]
You can modify `script.py.mako` and add the follow line:
```
import sqlmodel
```

**Reply by Kasuletrevor** [reply]
This helps first time. Thank you.

**Comment by KunxiSun** [comment]
First, Make sure you have the sqlmodel dependency
```python
pip install sqlmodel
```

Then, add this line in script.py.mako. That is how the sqlmodel author does in the [example](https://github.com/fastapi/full-stack-fastapi-template/blob/master/backend/app/alembic/script.py.mako). 
```python
import sqlmodel.sql.sqltypes
```

![image](https://github.com/user-attachments/assets/1417935b-7923-4492-bfe6-f71d009c1c44)

By adding this line, when you run `alembic revision ....`, the migration scripts will automatically have the `import import sqlmodel.sql.sqltypes`.
